cluster/get-kube.sh

```diff
@@ -177,15 +177,16 @@ release=${KUBERNETES_RELEASE:-"release/stable"}
 # Validate Kubernetes release version.
 # Translate a published version <bucket>/<version> (e.g. "release/stable") to version number.
 set_binary_version "${release}"
-if [[ ${KUBE_VERSION} =~ ${KUBE_RELEASE_VERSION_REGEX} ]]; then
-  release_url="${KUBERNETES_RELEASE_URL}/${KUBE_VERSION}/${file}"
-elif [[ ${KUBE_VERSION} =~ ${KUBE_CI_VERSION_REGEX} ]]; then
-  release_url="${KUBERNETES_CI_RELEASE_URL}/${KUBE_VERSION}/${file}"
-else
+if [[ ${KUBE_VERSION} =~ ${KUBE_CI_VERSION_REGEX} ]]; then
+  # Override KUBERNETES_RELEASE_URL to point to the CI bucket;
+  # this will be used by get-kube-binaries.sh.
+  KUBERNETES_RELEASE_URL="${KUBERNETES_CI_RELEASE_URL}"
+elif ! [[ ${KUBE_VERSION} =~ ${KUBE_RELEASE_VERSION_REGEX} ]]; then
   echo "Version doesn't match regexp" >&2
   exit 1
 fi
-
+export KUBERNETES_RELEASE_URL
```

Comment:
we need to set it in the environment for get-kube-binaries.sh to work properly for ci builds.

I reworked this a bit so we only inject it in the environment when calling get-kube-binaries.sh.